space-plugins/nuxt-base/app.config.ts

```diff
@@ -16,6 +19,10 @@ declare module '@nuxt/schema' {
 			errorCallback: string;
 			middleware?: {
 				ignoredPaths?: string[]; //e.g.: ['/api/endpoint']
+				afterAuth?: (params: {
```

Comment:
Hey @eunjae-lee 🙌 

Would be possible to create a type for this function? wdyt?